### PR TITLE
Products slider component

### PR DESCRIPTION
**Pull request by OmarSelim98**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+.package-lock.json
```
Comment (ZiyadKhalil) [inline .gitignore]: lock file shouldn't be ignored

Comment (OmarSelim98) [reply .gitignore]: ok

File: src/components/ProductCard/ProductCard.scss [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    .product-img{
+        padding-top: 10px;
+        width:90%;
+        height:auto;
+    }
+    .overlay{
+        position: absolute;
+        top:0;left:0;right:0;bottom:0;
+        border-style: solid;
+        border-width: 5px;
```
Comment (ZiyadKhalil) [inline src/components/ProductCard/ProductCard.scss]: my bad, border width is 1.5px check figma kda
transition should take 0.1s

File: src/components/ProductCard/ProductCard.scss [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        border-color: #f2c84b;
+        
+        opacity: 0;
+        transition: opacity 0.5s ease-in-out;
+        .slide-down-btn{
```
Comment (ZiyadKhalil) [inline src/components/ProductCard/ProductCard.scss]: what's this

File: src/components/ProductsSlider/ProductsSlider.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      <div className="slider-holder">
+        <ScrollMenu
+        data={menu}
+        arrowLeft={ArrowLeft}
+        arrowRight={ArrowRight}
+        menuClass="menu-class"
+        wrapperClass="wrapper-class"
+        dragging={false}
+        scrollBy={3}
+        alignCenter={false}
```
Comment (ZiyadKhalil) [inline src/components/ProductsSlider/ProductsSlider.js]: we need exactly 6 items aligned in center, no half product cards apparent on sides

File: src/components/ProductsSlider/ProductsSlider.scss [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+.card-style{
+  user-select: none;
+  user-select: none;
+  cursor: pointer;
+  border: none;
+  outline: none;
+  width: 14vw;
+  max-height:27.7vh;
+  padding-left: 0.625vw;
```
Comment (ZiyadKhalil) [inline src/components/ProductsSlider/ProductsSlider.scss]: ysta margins not paddings

File: src/components/ProductsSlider/ProductsSlider.scss [hunk 6]
```diff
@@ -0,0 +1,48 @@
+
```
Comment (ZiyadKhalil) [inline src/components/ProductsSlider/ProductsSlider.scss]: Sorry but hesham changed a bit in fonts size again, can you re-calculate them? also there's no item description anymore.
title is two lines and fuck ellipses msh mohm let's see how would it look (for now)

File: public/index.html [hunk 7]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script src="https://kit.fontawesome.com/f0d0617df8.js" crossorigin="anonymous"></script>
```
Comment (ZiyadKhalil) [inline public/index.html]: what's that doo boi

File: src/pages/HomePage/HomePage.js [hunk 8]
```diff
@@ -1,11 +1,90 @@
...
+}
+  ];
+
 function HomePage() {
     return (
         <div className="HomePage">
             <MainNavBar/>
             {/* {<Ping></Ping>} */}
+            
+            <ProductsSlider label="Best Deals" list={list}/>
```
Comment (ZiyadKhalil) [inline src/pages/HomePage/HomePage.js]: gamd

File: src/components/ProductCard/ProductCard.scss (hunk 3 above)
Comment (OmarSelim98) [reply src/components/ProductCard/ProductCard.scss]: idfk

File: .gitignore [hunk 9]
```diff
@@ -4,6 +4,7 @@
...
 
 # testing
 /coverage
```
Comment (OmarSelim98) [inline .gitignore]: eh

File: src/components/ProductsSlider/ProductsSlider.scss (hunk 6 above)
Comment (OmarSelim98) [reply src/components/ProductsSlider/ProductsSlider.scss]: done with ellipses *_*

File: public/index.html (hunk 7 above)
Comment (OmarSelim98) [reply public/index.html]: Icons and shit :/

**Pull request closed without merging**